tools/debian_kernel_postinst_lunatik.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
```

Comment:
I think you should use your own copyright ;-).. btw, can you also squash your commits? It helps to sanitize the history.. thanks!